src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,16 @@
-export const Tabs = () => {};
+export const Tabs = ({ tab, select, set }) => {
+  const check = tab.id === select.id;
```

Comment:
Comparing objects directly using `===` is not recommended as it can produce unexpected results and bugs. It's better to compare objects using their unique identifiers. In this case, you should compare `tab.id` with `select.id` instead of comparing `tab` with `select` directly.